css/devcenter.css

```diff
@@ -64,3 +64,101 @@ body,
   text-transform: uppercase;
   font-size: 80%;
 }
+
+/* New header css */
+
+.syn-page-header {
+  position: relative;
+}
+
+.syn-page-header-nav.mobile-nav-container {
+  height: calc(100vh - 100%);
+  left: 0;
+  overflow: auto;
+  position: absolute;
+  top: 100%; 
+  transform: translateX(0);
+  -webkit-transform: translateY(-100vh);
+  transform: translateY(-100vh);
+  width: 100%;
+}
+
+header.syn-page-header.nav-open .syn-page-header-nav.mobile-nav-container {
+  transform: translateX(0);
+}
+
+.open-break {
+  display: none;
+  position: absolute;
+  width: 100%;
+  height: 5px;
+  top: calc(100% + 2px);
+}
+
+.syn-dropdown.open .open-break {
+  display: block;
+}
+
+.syn-dropdown.open .syn-dropdown-list {
+  margin-top: 14px;
+  right: -50px;
+}
+
+.syn-page-header-content {
+ background-color: white;
+}
+
+.syn-styles-supported .display-inline {
+ display: inline; 
+}
+
+.syn-styles-supported .display-block {
+ display: block 
+}
+
+.syn-styles-supported .display-inline-flex {
```

Comment:
Is it necessary to scope these helper classes under `.syn-styles-supported`?

Reply:
For specificity. I'm having to override synapse classes with these helper classes. I suppose I could use `!important`, though I try not to.